### PR TITLE
Add documentation for output Analysis

### PR DESCRIPTION
**Pull request by niloofartrg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -471,7 +471,22 @@ We can combine the two precedent filters:
 The argument `-filter withtest` indicates that only commits with at least one modification on test cases are considered.
 
 
+## Analysis
```
Comment (sedflix) [inline README.md]: Hi @monperrus and @niloofartrg ,

Wouldn't it be better to have this documentation in the [README](https://github.com/kth-tcs/defects4j-repair-reloaded/blob/comrepair-coming/README.md) of [defects4j-repair-reloaded/comrepair-coming](https://github.com/kth-tcs/defects4j-repair-reloaded/blob/comrepair-coming/)?

WDYT?

**Comment by niloofartrg** [comment]
I reclone the project from GitHub, modify README file and then the checks fail in the PR.

apparently, the plugin vvvv can not be found.

Screen Shot 1398-04-28 at 11 41 25


Do you have any idea why? @monperrus @martinezmatias

File: README.md (hunk 1 above)
Comment (monperrus) [reply README.md]: It could be in both. There is never too much documentation in research prototypes :-)

**Comment by monperrus** [comment]
> the plugin vvvv can not be found.
this is exactly #116. Would you make a try to remove the spurious e.printStackTrace?

**Comment by niloofartrg** [comment]
checks for this PR are passed on the other documentation PR, everything seems fine there.

**Pull request closed without merging**